vms/event/views.py

```diff
@@ -28,12 +29,18 @@ def is_admin(request):
 
 @login_required
 def create(request):
+    
     if is_admin(request):
         if request.method == 'POST':
             form = EventForm(request.POST)
             if form.is_valid():
-                form.save()
-                return HttpResponseRedirect(reverse('event:list'))
+                start_date = form.cleaned_data['start_date']
+                if start_date < (datetime.date.today() - datetime.timedelta(days=1)):
+                    messages.add_message(request, messages.INFO, 'You can not create a post dated event')
```

Comment:
@vubo Let's give a more helpful error message to the end user :) Perhaps, "Start date should be tomorrow's date or later."